projects/packages/forms/src/blocks/contact-form/components/jetpack-integrations-modal/salesforce-card.tsx

```diff
@@ -66,12 +68,7 @@ const SalesforceCard = ( {
 		toggleDisabledTooltip: ! isValidSalesforceOrgId( salesforceData.organizationId )
 			? __( 'Enter a Salesforce Organization ID to enable.', 'jetpack-forms' )
 			: undefined,
-		setupBadge: (
-			<span className="integration-card__setup-badge">
-				<Icon icon="info-outline" size={ 12 } />
-				{ __( 'Enter organization ID', 'jetpack-forms' ) }
-			</span>
-		),
+		setupBadge: <Badge intent="info">{ __( 'Enter organization ID', 'jetpack-forms' ) }</Badge>,
```

Comment:
I updated this from 'default' to 'info'. Info is also gray, but includes the "i" icon, which keeps it consistent with other Badge intents that all have icons.

Reply:
I changed this back to default. See this comment for context: 
https://github.com/Automattic/jetpack/pull/44347#discussion_r2221044921